src/Differ/DiffConsoleFormatter.php

```diff
@@ -79,7 +79,7 @@ function ($string) use ($isDecorated, $lineTemplate) {
                     "#\n\r|\n#",
                     $isDecorated
                         ? OutputFormatter::escape(rtrim($diff))
-                        : $diff
+                        : rtrim($diff)
```

Comment:
This fixes a small inconsistency I noticed while fixing the test: without colored output, there is an extra line with three spaces after each diff. This does not occur with colored output.

Edit:

**before**
![before](https://cloud.githubusercontent.com/assets/1736542/25652831/56e9f79c-2fea-11e7-9627-6e37c18e657b.png)

**after**
![after](https://cloud.githubusercontent.com/assets/1736542/25652837/5ddc005e-2fea-11e7-8f42-18d1c26b2d66.png)